scrape.md

```diff
@@ -57,13 +57,13 @@ artoo.scrape('li', {id: 'id', content: 'text'});
   * **done**    *?function* : same as `callback`.
 * **callback**  *?function* : a function taking as only argument the scraped items and to be triggered when the scraping is done.
 
-Alternatively, you can pass a single object as argument to the `scrape` method and taking as properties `iterator`, `data` and `params`.
+Alternatively, you can pass a single object as an argument to the `scrape` method and setting the properties `iterator`, `data` and `params`.
 
 ---
 
 ### Data model
 
-Choosing a data model when using `artoo.scrape` is just a matter of deciding whether you want the function to return an array of values or rather an array of items with properties you prealably chose.
+Choosing a data model when using `artoo.scrape` is just a matter of deciding whether you want the function to return an array of values or rather an object with items and properties you designate.
```

Comment:
Artoo can return either an array of strings (can it return non-strings?)
['text of first li', 'text of second li']

or an array of objects containing  containing objects with properties designated by you.
[
  {text: 'text of fist li', id: 'first-li'},
  {text: 'text of second li', id: 'second-li'}
]